upstairs/src/test.rs

```diff
@@ -2173,10 +2174,10 @@ mod up_test {
         assert_eq!(state, AckStatus::Acked);
     }
 
-    // ZZZ another test that does read replay for the first IO
+    // XXX another test that does read replay for the first IO
```

Comment:
Ah, this was my not to self to make another test for some specific thing.  I'll go figure out what I was talking about and add the test and delete the comment.